app/stats/[slug]/csv/route.ts

```diff
@@ -0,0 +1,39 @@
+import { usePapaParse } from "react-papaparse";
```

Comment:
Je dobře, žes použila knihovnu – i u tak jednoduchých formátů, jako je CSV, je hodně jednoduché střelit se do nohy tím, že to generuješ „ručně“ pomocí řetězcových operací. Jako třeba já v jiných endpointech 😆 Zmiňuju to jednak abych podpořil to použití knihovny, jednak že bychom asi měli založit ticket pro pozdější přepis všech ostatních endpointů do téhle knihovny, ať to sjednotíme.

Reply:
(Jen je pro mě strašné WTF, co má parsování nebo renderování CSV společného s Reactem, že se ta knihovna jmenuje takhle, ale do toho se nebudu pouštět :)